src/courses/advanced/05.md

```diff
@@ -44,26 +44,49 @@ tar xzvf resources/git_test_target.tar.gz
 
 This will generate a `git_test` repository which we will use for these examples.
 
-Now let's write some tests and confirm that they run. You can put these tests in the `example.rb` file generated in the `controls` folder of your `my_git` InSpec profile. These tests are written using the `command` resource which is provided by InSpec. We will write a `git` resource in this section to improve this test. **Note that you will need to put the full directory path of the `.git` file from your `git_test_target` repository as the `git_dir` value on line 4 of `example.rb`. To get the full path of your current location in the terminal, use `pwd`.**
+Now let's write some tests and confirm that they run. You can put these tests in the `example.rb` file generated in the `controls` folder of your `my_git` InSpec profile. These tests are written using the `command` resource which is provided by InSpec. We will write a `git` resource in this section to improve this test.
 
-```ruby
-# encoding: utf-8
-# copyright: 2018, The Authors
+::: note Input review
+You will need to put the full directory path of the `.git` file from your `git_test_target` repository as the value for the `git_dir` input in your `inspec.yml`. To get the full path of your current location in the terminal, use `pwd`.  The value in the codeblock below should be correct, but it's always good to doublecheck.
+:::
```

Comment:
I think it would be beneficial to set a generic path in `inspec.yml` and an env-specific path in `inputs.yml` to align with what we teach.

```suggestion
::: note Input review
You will need to declare a `git_dir` input in your `inspec.yml` with a generic value such as `./.git`. You can then put the full directory path to the `.git` folder under your `git_test_target` repository as the value for the `git_dir` input in your `inputs.yml`. To get the full path of your current location in the terminal, use `pwd`.  The value in the code block below should be correct, but it's always good to doublecheck.
:::
```

Reply:
I don't want the students to have to add the --input-file flag every time that they run the profile.  I'll change up the language to say that we made this an input that is exactly specified for now and ought to be correct for the lab, but if it ain't then you can use the --input flag or an inputfile to overwrite it and also add the bit about a general location.